src/gluonnlp/vocab/bert.py

```diff
@@ -70,25 +78,30 @@ class BERTVocab(Vocab):
         A token used to separate sentence pairs for BERT.
     cls_token : hashable object or None, default '[CLS]'
         Classification symbol for BERT.
-    reserved_tokens : list of hashable objects or None, default [mask_token, sep_token, cls_token],
-        A list of reserved tokens (excluding `unknown_token`) that will always be indexed, such as
-        special symbols representing padding, beginning of sentence, and end of sentence. It cannot
-        contain `unknown_token` or duplicate reserved tokens. Keys of `counter`, `unknown_token`,
-        and values of `reserved_tokens` must be of the same hashable type. Examples: str, int, and
-        tuple.
+    reserved_tokens : list of hashable objects or None, default None
+        A list specifying additional reserved tokens that will always be
+        indexed similar to `padding_token`, `bos_token` or `eos_token`.
+        `reserved_tokens` cannot contain `unknown_token` or duplicate reserved
+        tokens.
```

Comment:
what does it mean "indexed similar to other tokens"?

Reply:
the way that other tokens are indexed are not documented either.

Reply:
This should clarify "such as special symbols representing padding, beginning of sentence, and end of sentence" which was used before. Let's see if it can be made more clear.

Reply:
https://github.com/dmlc/gluon-nlp/pull/750